lib/timeline/locales.js

```diff
@@ -47,22 +51,34 @@ exports['fr_BE'] = exports['fr'];
 require('moment/locale/es');
 exports['es'] = {
   current: 'corriente',
-  time: 'hora'
+  time: 'hora',
+  deleteSelected: 'Eliminar selecci\u00f3n',
```

Comment:
I don't think you need the escape character her. Unicode is supported: `Eliminar selección`